cmd/create.go

```diff
@@ -35,6 +34,10 @@ var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create the resource on the Kubernetes cluster",
 	Run: func(cmd *cobra.Command, args []string) {
+		if err := ifFilesPassed(CreateFiles); err != nil {
+			fmt.Println(err)
```

Comment:
@kadel to print the help, I need to add `createCmd.Help()` here, but cobra won't allow and ends up throwing an initialization error. Not sure how to work around that, maybe we can do that later and just print the error for now?